precompile/contracts/warp/contract.go

```diff
@@ -31,12 +32,17 @@ const (
 	SendWarpMessageGasCost uint64 = contract.LogGas + 3*contract.LogTopicGas + AddWarpMessageGasCost + contract.WriteGasCostPerSlot
 	// SendWarpMessageGasCostPerByte cost accounts for producing a signed message of a given size
 	SendWarpMessageGasCostPerByte uint64 = contract.LogDataGas
-
-	GasCostPerWarpSigner            uint64 = 500
-	GasCostPerWarpMessageChunk      uint64 = 3_200
-	GasCostPerSignatureVerification uint64 = 200_000
 )
 
+type GasConfig struct {
+	// Gas cost per warp signer in the validator set
+	PerWarpSigner uint64
+	// Gas cost per chunk of the warp message (each chunk is 128 bytes)
+	PerWarpMessageChunk uint64
+	// Gas cost to verify a warp message signature
```

Comment:
```suggestion
	// Gas cost to verify a BLS signature
```

I think of the "warp message signature" as the BLS aggregate signature and bit vector together, as opposed to this just being the fixed cost to verify a BLS signature given a public key.